src/openloco/windows/construction/Construction.h

```diff
@@ -0,0 +1,377 @@
+    static loco_global<previewTrack[4], 0x004F7807> _unk_4F7807;
+    static loco_global<previewTrack[1], 0x004F7830> _unk_4F7830;
+    static loco_global<previewTrack[1], 0x004F783B> _unk_4F783B;
+    static loco_global<previewTrack[1], 0x004F7846> _unk_4F7846;
+    static loco_global<previewTrack[1], 0x004F7851> _unk_4F7851;
+    static loco_global<previewTrack[1], 0x004F785C> _unk_4F785C;
+    static loco_global<previewTrack[1], 0x004F7867> _unk_4F7867;
+    static loco_global<previewTrack[1], 0x004F7872> _unk_4F7872;
+    static loco_global<previewTrack[1], 0x004F787D> _unk_4F787D;
+    static loco_global<previewTrack[1], 0x004F7888> _unk_4F7888;
+    static loco_global<previewTrack[1], 0x004F7893> _unk_4F7893;
+    static loco_global<previewTrack[1], 0x004F789E> _unk_4F789E;
+    static loco_global<previewTrack[1], 0x004F78A9> _unk_4F78A9;
+    static loco_global<previewTrack[1], 0x004F78B4> _unk_4F78B4;
+    static loco_global<previewTrack[1], 0x004F78BF> _unk_4F78BF;
+    static loco_global<previewTrack[1], 0x004F78CA> _unk_4F78CA;
+    static loco_global<previewTrack[1], 0x004F78D5> _unk_4F78D5;
+    static loco_global<previewTrack[1], 0x004F78E0> _unk_4F78E0;
+    static loco_global<previewTrack[1], 0x004F78EB> _unk_4F78EB;
+
+    static loco_global<uint16_t[351][4], 0x004F7B62> _word_4F7B62; // TODO: Not sure on size?
+    static loco_global<uint8_t[31], 0x005045FA> _byte_5045FA;
+    static loco_global<uint8_t, 0x00508F09> _byte_508F09;
+    static loco_global<uint8_t, 0x00522090> _byte_522090;
+    static loco_global<uint8_t, 0x00522091> _byte_522091;
+    static loco_global<uint8_t, 0x00522092> _byte_522092;
+    static loco_global<uint8_t, 0x00522095> _byte_522095;
+    static loco_global<uint8_t, 0x00522096> _byte_522096;
+    static loco_global<uint16_t, 0x0052338A> _tooltipTimeout;
+    static loco_global<ui::window_number, 0x00523390> _toolWindowNumber;
+    static loco_global<ui::WindowType, 0x00523392> _toolWindowType;
+    static loco_global<uint32_t, 0x00523394> _toolWidgetIndex;
+    static loco_global<company_id_t, 0x00525E3C> _playerCompany;
+    static loco_global<uint8_t[8], 0x0525F72> _scenarioSignals;
+    static loco_global<uint8_t[8], 0x0525F7A> _scenarioBridges;
+    static loco_global<uint8_t[8], 0x0525F82> _scenarioTrainStations;
+    static loco_global<uint8_t[8], 0x0525F8A> _scenarioTrackMods;
+    static loco_global<uint8_t[8], 0x0525F9A> _scenarioRoadStations;
+    static loco_global<uint8_t[8], 0x0525FA2> _scenarioRoadMods;
+    static loco_global<uint8_t, 0x00525FAA> _lastRailroadOption;
+    static loco_global<uint8_t, 0x00525FAB> _lastRoadOption;
+    static loco_global<uint8_t, 0x00525FAC> _lastAirport;
+    static loco_global<uint8_t, 0x00525FAD> _lastShipPort;
+    static loco_global<uint8_t, 0x00525FAE> _byte_525FAE;
+    static loco_global<company_id_t, 0x009C68EB> _updatingCompanyId;
+    static loco_global<uint32_t, 0x00E0C3E0> _dword_E0C3E0;
```

Comment:
This is clearly a `gfx::drawpixelinfo_t*`. Just change it to that type then you don't need the casts.